### PR TITLE
Adding custom layout support

### PR DESCRIPTION
**Pull request by evgeniyraev**
adding custom layouts
changing layouts to be more generic
removed specific layouts names used in Keyboard

### CODE REVIEW

**Comment by WiaczeslawP** [comment]
Could you rebase from master branch and update your code according to eslint config and squash your commits?

File: src/Keyboard.js [hunk 1]
```diff
@@ -35,14 +34,19 @@ export default class Keyboard extends PureComponent {
 		this.handleSymbolsClick = this.handleSymbolsClick.bind(this);
 
 		this.state = {
-			currentLanguage: props.defaultLanguage,
+			currentLanguage: 0,
+			languages: props.languages ==null ? [LatinLayout,CyrillicLayout] : this.props.languages,
```
Comment (WiaczeslawP) [inline src/Keyboard.js]: `languages` value never changes inside component, so there is no sense to place it in the state. 
I would place this value in `props`, and `[LatinLayout,CyrillicLayout]` is its defaultProp

File: src/layouts/CyrillicLayout.js [hunk 2]
```diff
@@ -1,5 +1,9 @@
-export default [
-	['й', 'ц', 'у', 'к', 'е', 'н', 'г', 'ш', 'щ', 'з', 'х', 'ъ'],
-	['ф', 'ы', 'в', 'а', 'п', 'р', 'о', 'л', 'д', 'ж', 'э'],
-	['я', 'ч', 'с', 'м', 'и', 'т', 'ь', 'б', 'ю'],
-];
+export default {
```
Comment (WiaczeslawP) [inline src/layouts/CyrillicLayout.js:1]: Can you add layouts to the export in the main file?  https://github.com/WiaczeslawP/react-screen-keyboard/blob/master/Keyboard.js

**Comment by evgeniyraev** [comment]
I dont know your build configuration. If you add you build scripts/commands I can probably do that

**Comment by evgeniyraev** [comment]
Can you tell me how to test it?
For my change I added the keyboard to existing project and test it there.

**Comment by WiaczeslawP** [comment]
`npm run lint` in your terminal will show errors

**Comment by WiaczeslawP** [comment]
What do you mean? Unit testing? There isn't any written tests

**Comment by evgeniyraev** [comment]
I suggest to move conversation briefly in slack
https://react-screen-keyboard.slack.com/shared_invite/MTQ1OTg4Mjc1MTU1LTE0ODc5NDcxMTItOTUwN2QyZjI4OQ

File: src/Keyboard.js [hunk 3]
```diff
@@ -23,7 +22,7 @@ export default class Keyboard extends PureComponent {
 		leftButtons: [],
 		rightButtons: [],
 		isFirstLetterUppercase: false,
-		defaultLanguage: 'cyrrilic',
+		languages: [LatinLayout, CyrillicLayout],
```
Comment (WiaczeslawP) [inline src/Keyboard.js]: `static propTypes` should be editted too

Comment (WiaczeslawP) [reply src/Keyboard.js]: The default layout was cyrillic, can you place `CyrillicLayout` on the `0` pisition?

**Pull request merged**